build/yamls/base/controller.yml

```diff
@@ -18,6 +18,19 @@ metadata:
 ---
 apiVersion: apiregistration.k8s.io/v1
 kind: APIService
+metadata:
+  name: v1beta2.controlplane.antrea.tanzu.vmware.com
+spec:
+  group: controlplane.antrea.tanzu.vmware.com
+  groupPriorityMinimum: 100
+  version: v1beta2
+  versionPriority: 100
```

Comment:
I know that when `versionPriority` is the same, it breaks ties based on the semantic version number, so priority of `v1beta2` is greater than priority of `v1beta1`

Reply:
```
In case of equal version priorities, the version string will be used to compute the order inside a group.
If the version string is "kube-like", it will sort above non "kube-like" version strings, which are ordered
lexicographically. "Kube-like" versions start with a "v", then are followed by a number (the major version),
then optionally the string "alpha" or "beta" and another number (the minor version). These are sorted first
by GA > beta > alpha (where GA is a version with no suffix such as beta or alpha), and then by comparing 
major version, then minor version. An example sorted list of versions:
 v10, v2, v1, v11beta2, v10beta3, v3beta1, v12alpha1, v11alpha2, foo1, foo10.
```
https://godoc.org/k8s.io/kube-aggregator/pkg/apis/apiregistration/v1beta1